src/App.tsx

```diff
@@ -10,9 +12,18 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/people">
+              <Route index element={<PeoplePage />} />
+              <Route path=":slug" element={<PeoplePage />} />
+            </Route>
+            <Route path="home" element={<Navigate to="/" replace />} />
+            <Route
+              path="*"
+              element={<h1 className="title">Page not found</h1>}
+            />
```

Comment:
The wildcard route (`path="*"`) is correctly set up to catch all undefined paths and display a 'Page not found' message. However, ensure that this route is placed at the end of all other routes to prevent it from catching paths that should be handled by other routes. In this case, it is already at the end, so no changes are needed.